Dockerfile

```diff
@@ -54,6 +54,18 @@ RUN install bin/fluent-bit /fluent-bit/bin/
 # Configuration files
 COPY fluent-bit.conf \
      /fluent-bit/etc/
+COPY ecs/firehose.conf \
+     /ecs/
+COPY ecs/cloudwatch.conf \
+     /ecs/
+COPY ecs/kinesis.conf \
+     /ecs/
+COPY ecs/amazon-es.conf \
+     /ecs/
+COPY ecs/s3.conf \
+     /ecs/
+COPY ecs/filter-parser.conf \
+     /ecs/
```

Comment:
Wondering can't we just copy the whole `/ecs` folder in one command?

Reply:
good point. Let me explore on it

Reply:
Also, I prefer to move all these files under `config/ecs/`.

Reply:
I don't like it... it makes the path longer. Remember that folks will specify the full path in our API- I want it to be as short as possible

Reply:
Thinking if we consider adding default configs for EKS and other platforms also. In that case all the configs under `/config` folder might be considered as good design.

Reply:
For EKS there is not a strong need I think, config maps exist and are super easy to use. 

This work is just for our new project...

Reply:
If so, why do we need to mention `/ecs`. We may put everything under `/config` folder.

Reply:
I am OK with either of them. Let's move with our original plan (only `/ecs`).